src/polyfem/assembler/RhsAssembler.cpp

```diff
@@ -626,18 +626,17 @@ namespace polyfem
 				for (int i = 0; i < lb.size(); ++i)
 				{
 					const int primitive_global_id = lb.global_primitive_id(i);
-					const auto nodes = bs.local_nodes_for_primitive(primitive_global_id, mesh_);
```

Comment:
why changing?

Reply:
It's best to keep the declaration before the use of the variables, improves the readability.